source/renderer/app/i18n/locales/ja-JP.json

```diff
@@ -1187,7 +1187,7 @@
   "wallet.summary.transactionsList.syncingTransactionsMessage": "ウォレットのトランザクション履歴は現在ブロックチェーンと同期中です。",
   "wallet.summary.transactionsList.todayLabel": "今日",
   "wallet.summary.transactionsList.yesterdayLabel": "昨日",
-  "wallet.token.picker.addButtonLabel": "キャンセル",
+  "wallet.token.picker.addButtonLabel": "追加する",
```

Comment:
Tiny fix to address the issue from user report: https://github.com/input-output-hk/daedalus/issues/2931
Slack thread: https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1648039378845489, https://input-output-rnd.slack.com/archives/CBECZ2K47/p1648025188531639